paper_framework_sources/p2079_system_execution_context.md

```diff
@@ -20,7 +20,6 @@ A `system_context` and `system_scheduler` that expose a simple parallel-forward-
 # Changes
 ## R2
 - Significant redesign to fit in P2300 model.
-- Remove priorities.
```

Comment:
Since we didn't have priorities in previous revisions we don't "remove" them in R2, we just don't add them:)

Reply:
Ah, good point :D